bcgov/HOUS-permit-portal
# Show correct application ID in PDF footer

## Description

Show application number instead of UUID in PDF footer

## What type of PR is this? (check all applicable)


## Related Tickets & Documents

[BPHH-1116](https://hous-bssb.atlassian.net/browse/BPHH-1116)

## Steps to QA

1. Submit permit application with step code checklist
2. Download the application files as a reviewer
3. Verify the application form and step code PDFs use the permit application number in the footer

## UI Accessibility Checklist



_Note the following checklist is not an exhaustive list. Check
the [WebAIM checklist](https://webaim.org/standards/wcag/checklist) for
a user-friendly checklist with more details. Aim to meet at least *AA* conformance level where applicable. Check
the [WCAG guidelines](https://webaim.org/standards/wcag/checklist) for the full guidelines._

      with [Wave Browser Extension](https://chromewebstore.google.com/detail/wave-evaluation-tool/jbbplnpkjmmeebjpijfedlgcdilocofh)
      for accessibility errors?

## General Checklist


## [optional] Are there any post-deployment tasks we need to perform?

n/a
